x-pack/plugins/security_solution/cypress/integration/network/overflow_items.spec.ts

```diff
@@ -13,26 +13,30 @@ import {
   FILTER_OUT,
   SHOW_TOP_FIELD,
 } from '../../screens/network/flows';
-import { cleanKibana } from '../../tasks/common';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
 
-import { loginAndWaitForPage } from '../../tasks/login';
+import { login, visit } from '../../tasks/login';
 import { openHoverActions } from '../../tasks/network/flows';
 
 import { NETWORK_URL } from '../../urls/navigation';
 
-const testDomainOne = 'endpoint-dev-es.app.elstc.co';
-const testDomainTwo = 'endpoint2-dev-es.app.elstc.co';
+const testDomainOne = 'myTest';
+const testDomainTwo = 'myTest2';
 
 describe('Overflow items', () => {
   context('Network stats and tables', () => {
     before(() => {
-      cleanKibana();
+      esArchiverLoad('network');
+      login();
     });
     beforeEach(() => {
-      loginAndWaitForPage(NETWORK_URL);
+      visit(NETWORK_URL);
+    });
+    after(() => {
+      esArchiverUnload('network');
     });
 
-    it('Shows more items in the popover', () => {
+    it.skip('Shows more items in the popover', () => {
```

Comment:
Why is it skipped? Could you leave a comment with an explaination?

Reply:
It has been skipped by mistake. So lots lots of thanks for pointing this.